src/java/README.md

```diff
@@ -0,0 +1,135 @@
+## Java Language Basic Syntax Review
+Java is a very popular general-purpose programming language, it is class-based and object-oriented. Java was developed by James Gosling at Sun Microsystems ( later acquired by Oracle) the initial release of Java was in 1995. Java 17 is the latest long-term supported version (LTS). As of today, Java is the world's number one server programming language with a 12 million developer community, 5 million students studying worldwide and it's #1 choice for the cloud development.
+
+Key features:
+
+- Well maintained
+- Object-oriented
+- Robust and Secure
+- Multithreading
+- Distributed Computing
+- Dynamic and Extensible 
+- Owned by oracle. Positive or negative you decide.
+
+#### 1. Read Inputs
+
+````java 
+import java.util.Scanner;
+class Input {
+    public static void main(String[] args) {
+    	Scanner input = new Scanner(System.in);
+    	System.out.println("Enter your name: ");
+    	String inp = input.next();

```

Comment:
perhaps update this to input.nextLine()?
Or line 21 to 'Enter your first name:' 
input.next() will only read the next token, and if someone enters their last name for instance, this will not read it entirely.